Cairo Desktop/CairoDesktop.Common/Localization/DisplayString.cs

```diff
@@ -1,720 +1,857 @@
+                {
+                    // Ignore empty lines and comments (lines starting with #)
+                    if (string.IsNullOrWhiteSpace(line) || line.StartsWith("#"))
+                        continue;
+
+                    // Split the line into key and value based on '='
+                    
+                    string[] parts = line.Split('=');
+                    
+                    if (parts.Length == 2)
+                    {
+                        string key = parts[0].Trim();
+                        string value = parts[1].Trim();
+
+                        // Add to dictionary if the key is not empty
+                        if (!string.IsNullOrEmpty(key))
+                            dictionary[key] = value;
+                    }
+                }
+            }
+            else
+            {
+                Console.WriteLine($"File not found: {filePath}");
+            }
+
+            return dictionary;
+        }
+
+        private static string getString([CallerMemberName] string stringName = null)
+        {           
+            bool isDefault = false;
+            string useLang = Settings.Instance.Language.ToLower();
+            /*
+            if (useLang.StartsWith("fr_"))
+            {
+                lang = Language.fr_FR;
+            }
+            else if (useLang.StartsWith("it_"))
+            {
+                lang = Language.it_IT;
+            }
+            else if (useLang.StartsWith("pt_"))
+            {
+                lang = Language.pt_BR;
+            }
+            else if (useLang.StartsWith("sv_"))
+            {
+                lang = Language.sv_SE;
+            }
+            else if (useLang.StartsWith("zh_"))
+            {
+                lang = Language.zh_CN;
+            }
+            else if (useLang.StartsWith("cs_"))
+            {
+                lang = Language.cs_CZ;
+            }
+            else if (useLang.StartsWith("de_"))
+            {
+                lang = Language.de_DE;
+            }
+            else if (useLang.StartsWith("es_"))
+            {
+                lang = Language.es_ES;
+            }
+            else if (useLang.StartsWith("nl_"))
+            {
+                lang = Language.nl_NL;
+            }
+            else if (useLang.StartsWith("ru_"))
+            {
+                lang = Language.ru_RU;
+            }
+            else if (useLang.StartsWith("pl_"))
+            {
+                lang = Language.pl_PL;
+            }
+            else if (useLang.StartsWith("tr_"))
+            {
+                lang = Language.tr_TR;
+            }
+            else if (useLang.StartsWith("ko_"))
+            {
+                lang = Language.ko_KR;
+            }
+            else if (useLang.StartsWith("hu_"))
+            {
+                lang = Language.hu_HU;
+            }
+
+            else
+            {
+                lang = Language.en_US;
+                isDefault = true;
+            }*/

```

Comment:
No reason to keep all of the old code around, we have it in git history :)

Reply:
Removed, was just kept if something doesn't work so I can check what I did.